core/sbt-exercise/src/main/scala/com/fortysevendeg/exercises/sbtexercise/ExerciseCompilerPlugin.scala

```diff
@@ -194,6 +194,8 @@ object ExerciseCompilerPlugin extends AutoPlugin {
         }
       }
 
+    def relativePath(absolutePath: String, splitter: String) = splitter + absolutePath.split(splitter)(1)
```

Comment:
Technically `.apply(1)` could throw an unchecked exception, if the splitter character wasn't found in the path.